automated-comments/go/hamming/comparing_bytes_disguised_as_runes.md

```diff
@@ -0,0 +1,5 @@
+Converting a `byte` to a `rune` works perfectly in the context of this exercise as there are only ASCII letters in DNA strands.
+ASCII characters consist of one `byte` each. As soon as non-ASCII characters get involved this will break as they consist of 
+up to 4 bytes.
```

Comment:
Should this line be combined with the previous one? (Not sure how whitespace is treated in the UI)

Reply:
New lines are mark-down new lines, so a treated as a space if there's only one.